### PR TITLE
Dockerfile 추가

### PR DESCRIPTION
**Pull request by yunochi**
므아한 compose.yml 도 추가

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM debian:bookworm-slim
+
+RUN mkdir -p /opt/fedi-killspam
+RUN mkdir -p /config
```
Comment (tribela) [inline Dockerfile]: I think `VOLUME` command is better for this situation

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+WORKDIR /opt/fedi-killspam
+
+
+RUN apt-get update && apt-get install -y curl unzip
+
+ADD . /opt/fedi-killspam/
+RUN rm -f /opt/fedi-killspam/config.json && ln -s /config/config.json /opt/fedi-killspam/config.json
```
Comment (tribela) [inline Dockerfile]: Can use `.gitignore` to not add config.json

Comment (yunochi) [reply Dockerfile]: 아 dockerignore가 있었지!

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
+FROM debian:bookworm-slim
+
+RUN mkdir -p /opt/fedi-killspam
+VOLUME /config
+WORKDIR /opt/fedi-killspam
+
+
+RUN apt-get update && apt-get install -y curl unzip
+
+ADD . /opt/fedi-killspam/
```
Comment (tribela) [inline Dockerfile]: `COPY` is more prefered way to add source code

FYI: https://phoenixnap.com/kb/docker-add-vs-copy

**Comment by nulta** [comment]
LGTM!

**Pull request merged**